go.mod

```diff
@@ -113,3 +113,5 @@ require (
 )
 
 go 1.20
+
+replace github.com/codeready-toolchain/api => github.com/filariow/toolchain-api v0.0.0-20240430081956-c9bf496de75d
```

Comment:
Let's move it to a separate PR. See the reasoning here: https://redhat-internal.slack.com/archives/C06MJ2DVBU4/p1713987983373619?thread_ts=1713955118.062449&cid=C06MJ2DVBU4

Reply:
+1